core/domain/story_domain_test.py

```diff
@@ -102,6 +102,19 @@ def test_get_acquired_skill_ids_for_node_ids(self):
             ['skill_1', 'skill_2']
         )
 
+    def test_get_acquired_skill_ids_for_node_ids_special_cases(self):
+        # Test cases when there are multiple acquired skill ids linked to
```

Comment:
Test each behaviour/case in a separate test. If you need to describe your tests using "A and B" or "special cases", this means you need to split some tests up. Each test should be single-purpose.

Reply:
Done.